README.md

```diff
@@ -26,31 +26,51 @@ Next follow these instructions.
     PASSWORD: osp
     HOST: localhost
     ``` 
-    You may run the following commands for local setup of DB:
+    You may run the following commands for local setup of DB in Linux:
 
     ```
+    cd open-source-programs-backend
     sudo -i -u postgres
     createuser osp --pwprompt
     psql
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+2. You need to download API key file from your user-settings on Zulip. The file you download is named as 'download' or rename that to 'download'.
+3. Place that download file in the project's directory.
 
-
-2. To start the server:
+4. Move into the project's directory.
 
     ```
     cd open-source-programs-backend
+    ```
+5. Create virtual environment (this is not a hard requirement, but its advisable)
+    ```
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
+    ```
+6. To run the migrations run: 
+   ```
+   python manage.py migrate
+   ```
```

Comment:
```suggestion
   ``python manage.py migrate``
```

@isabelcosta shouldn't this be changes like this

Reply:
why @devkapilbansal ? I am not sure what "``" does 🤔 I am really curious
I think like it is, its ok

Reply:
It is generally used in multiline codes. For single line code, single backquote is used as per conventions.